drizzlepac/haputils/poller_utils.py

```diff
@@ -426,13 +445,23 @@ def parse_mvm_tree(det_tree, log_level):
                 #
                 # svm prod_info = 'skycell_p1234_x01y01 wfc3 uvis f200lp all 2009 1 drz'
                 #
+
                 prod_list = prod_info.split(" ")
-                layer = (prod_list[3], prod_list[4], prod_list[5])
-                ftype = prod_list[-1]
+                pscale = 'fine' if prod_list[2] != 'IR' else 'coarse'
```

Comment:
prod_list is derived from prod_info which is lower case. The comparison to 'IR' is not correct.

Reply:
Resolved in 21fa202.